packages/rich-text-editor/src/EditableRichTextContent/EditableRichTextContent.module.scss

```diff
@@ -19,10 +19,17 @@
   border-color: $color-blue-500;
 }
 
-// Temporary solution for focus styling the content area
-// (the problem with this is that it shows this styling when focusing a link within the content
-.editableContainer:focus-within :global(.ProseMirror) {
-  background-color: $color-gray-200;
+.editableContainer .hiddenButton {
+  clip: rect(0 0 0 0);
+  clip-path: inset(50%);
+  height: 1px;
+  overflow: hidden;
+  position: absolute;
+  white-space: nowrap;
+  width: 1px;
+}
+
+.editableContainer .hiddenButton:focus + * > :global(.ProseMirror) {
```

Comment:
This work but keen to hear opinions of this going too far into the route of scss hackery (or sackery for short)

Reply:
Oh nice! I forgot that it was the sibling that needed the focus styling rather than the parent.

I'm happy with this.